src/libexpr/primops.cc

```diff
@@ -4133,26 +4308,46 @@ void EvalState::createBaseEnv()
         attrs.alloc("prefix").mkString(i.first);
         (v.listElems()[n++] = allocValue())->mkAttrs(attrs);
     }
-    addConstant("__nixPath", v);
+    addConstant("__nixPath", v, {
+        .type = nList,
+        .doc = R"(
+          The search path used to resolve angle bracket path lookups.
+
+          Angle bracket are
```

Comment:
```suggestion
          Angle bracket expressions are
```

We direly need an authoritative name and an anchor in the language reference for that.